### PR TITLE
Inference Shortcuts + CoremelysisML framework

### PR DESCRIPTION
**Pull request by Galdineris**
## Motivation
One of our main goals with Coremelysis was making use of mobile features to enhance 
experience sampling techniques. With that in mind, we planned on using Siri Shortcuts to
provide native automation to the process.

## Modifications
- New CoremelysisML framework created.
- MLManager, Sentiment, SentimentAnalysisModel, MachineLearningError and SentimentPolarity files all moved to CoremelysisML.
- CoremelysisShortcutsIntents extension created.

## Result
Siri Shortcuts and, so Shortcuts app, can now be used to create task flows with our app.

### CODE REVIEW

File: Coremelysis/Coremelysis/MainView/MainViewModel.swift [hunk 1]
```diff
@@ -3,10 +3,12 @@
 //  Coremelysis
 //
 //  Created by Artur Carneiro on 18/08/20.
-//  Copyright © 2020 Rafael Galdino. All rights reserved.
+//  Copyright © 2020 Rafael Galdino. All rights rSwift Compiler Warning Groupeserved.
```
Comment (csfar) [inline Coremelysis/Coremelysis/MainView/MainViewModel.swift:6]: 😂

Comment (csfar) [reply Coremelysis/Coremelysis/MainView/MainViewModel.swift:6]: Don't worry, we fix it in some later PR.

**Review by csfar — APPROVED** [review]
chefsKiss.gif

File: Coremelysis/Coremelysis/MainView/MainViewModel.swift (hunk 1 above)
Comment (Galdineris) [reply Coremelysis/Coremelysis/MainView/MainViewModel.swift:6]: hahaha must have happened during one of the merges.

**Pull request merged**